thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -0,0 +1,31 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  entities: null, // {}
+
+  selectedUrns: null, // Set
+
+  onSelection: null, // function (Set, state)
+
+  labels: Ember.computed(
+    'entities',
+    function () {
+      const { entities } = this.getProperties('entities');
+
+      const labels = {};
+      Object.keys(entities).forEach(urn => labels[urn] = entities[urn].label);
```

Comment:
nit: we can use a reduce function since it's a n to 1 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce?v=a